minfraud/models.py

```diff
@@ -296,13 +296,31 @@ class Device(object):
       RFC 3339 date-time.
 
       :type: str | None
+
+    .. attribute:: session_age
```

Comment:
These are the response models. There are not models for the request. Instead we do validation on a dict:

https://github.com/maxmind/minfraud-api-python/blob/9b15e69971bd27b9a5dc4282df78de5147ab0504/minfraud/validation.py#L230

This code should be removed.